### ISSUE TITLE
[2.1] Volt cannot iterate on config object.

### ISSUE THREAD

**Issue by dschissler** [phalcon/cphalcon#11356]
I want to iterate like this: 
```volt
server_name{% for domain in domains %} {{ domain }}{% endfor %};
```
and `domains` is being set from a config object and it doesn't work on Phalcon 2.1.  The work around is to convert to an array first with 
```php
$config->site->domains->toArray();
```

**Comment by sergeyklay** [comment]
Coud you please provide object?

**Comment by dschissler** [comment]
Yes sir.  A nice little collection of files could also help to expose the limitations in what I will call the "stand alone" compiler.  Yesterday in my public project I found a situation in which `{{ request... }}` works in the raw compiled templates but not in the precompiled ones so I had to change it to `{{ this.request...}}`.  I will build a little stand alone test suite towards this other objective while testing this one.
